### DISCUSSION TITLE
VS 2022 - Android Generators

### DISCUSSION THREAD

**Discussion by Crocks74** [shinyorg/shiny#969] [Q&A] [answered]
While setting up a new XF Project, I installed the Shiny from NuGet 2.4.3.3024 and upon inspection I noticed under Android Analyzers the Shiny.Generator did not generate any files.  Am I missing something?

![image](https://user-images.githubusercontent.com/20951302/161111443-a0e38a98-d15d-490e-9b19-89012c70542b.png)

**Comment by aritchie** [comment, marked as answer]
The attributes - please read the docs as this is covered: https://shinylib.net/quickstart
